web/templates/web/admin_panel.html

```diff
@@ -92,6 +92,30 @@ <h1>{% trans "Admin panel" %}</h1>
                     </a>
                 </div>
             {% endif %}
+            {% if user|has_any_tools_permission or user.is_superuser %}
+                <div class="ui yellow card">
+                    <a href="{% url 'makerspace-tools-admin' %}" class="content">
+                        <div class="center aligned header">
+                            <i class="ui grey tools icon"></i>{% trans "Tools" %}
+                        </div>
+                        <div class="description">
+                            {% trans "Here you can add or edit tools at MAKE NTNU." %}
+                        </div>
+                    </a>
+                </div>
+            {% endif %}
+            {% if  user|has_any_makerspace_permission or user.is_superuser %}
+                <div class="ui yellow card">
```

Comment:
Is this card needed? It just links to the publicly available makerspace info page.